benchmark/src/runner.rs

```diff
@@ -82,6 +82,7 @@ impl Stats {
     /// Calculate the histogram from the latencies.
     #[allow(clippy::indexing_slicing)]
     #[allow(clippy::let_underscore_must_use)] // the writeln! will not fail thus always return an Ok
+    #[allow(clippy::arithmetic_side_effects)] // test only
```

Comment:
Should the comment here be "benchmark only"?

Reply:
done